website/docs/amazon-emr-on-eks/emr-eks-karpenter.md

```diff
@@ -2,32 +2,158 @@
+| Metrics server | Yes | Kubernetes metrics server is responsible for aggregating cpu, memory and other container resource usage within your cluster | [EKS Metrics Server Documentation](https://docs.aws.amazon.com/eks/latest/userguide/metrics-server.html) |
+| Prometheus | Yes | Prometheus is responsible for monitoring EKS cluster including spark applications in your EKS cluster. We use Prometheus deployment for scraping and ingesting metrics into Amazon Managed Prometheus and Kubecost | [Prometheus Documentation](https://prometheus.io/docs/introduction/overview/) |
+| Amazon Managed Prometheus | Yes | This is responsible for storing and scaling of EKS cluster and spark application metrics | [Amazon Managed Prometheus Documentation](https://docs.aws.amazon.com/prometheus/latest/userguide/what-is-Amazon-Managed-Service-Prometheus.html) |
+| Kubecost | Yes | Kubecost is responsible for providing cost break down by Spark application. You can monitor costs based on per job, namespace or labels | [EKS Kubecost Documentation](https://docs.aws.amazon.com/eks/latest/userguide/cost-monitoring.html) |
+| CloudWatch metrics | No | CloudWatch container insights metrics shows simple and standardized way to monitor not only AWS resources but also EKS resources on CloudWatch dashboard | [CloudWatch Container Insights Documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-metrics-EKS.html) |
+|AWS for Fluent-bit | No | This can be used to publish EKS cluster and worker node logs to CloudWatch Logs or 3rd party logging system | [AWS For Fluent-bit Documentation](https://github.com/aws/aws-for-fluent-bit) |
+| FSx for Lustre CSI driver | No | This can be used for running Spark application using FSx for Lustre | [FSx for Lustre CSI Driver Documentation](https://docs.aws.amazon.com/eks/latest/userguide/fsx-csi.html) |
+
+
+### Customizing Add-ons
+
+You can customize your deployment anytime either by changing recommended system add-ons in `addons.tf` or by changing optional add-ons in `variables.tf`. 
+
+For example, let's say you want to remove Amazon Managed Prometheus because you have another application that captures Prometheus metrics, you can edit `addons.tf` using your favorite editor, find Amazon Managed Prometheus and change to `false`
+```
+enable_amazon_prometheus             = false
+```
+
+If you want to use FSx for Lustre storage while running Spark application for storing shuffle files or accessing data from S3, you can install FSx CSI driver by searching for FSx in `variables.tf` and edit the file
+```
+variable "enable_aws_fsx_csi_driver" {
+  default     = true
+```
+Once the changes are saved, follow the [deployment guide](#deploying-the-solution) if this is a new installation or apply these changes using Terraform for existing installation
+```
+terraform apply
+```
+
+## Deploying the Solution
```

Comment:
Let's use the import CollapsibleContent from '../../src/components/CollapsibleContent'; feature for  `Deploying the Solution`. This page has too much content and we need improve the look and feel. Same for the `Cleanup` as well

Reply:
I'm using markdown method for toggling. If you look for provisioners, i am toggling the code using <details> ... </details>. Seems much easier to do rather than using CollapsibleContent
https://gist.github.com/Ashton-W/864d09d71f7cb8e5b47ebaac02b98455